catroid/src/embroideryDesigner/res/values/colors.xml

```diff
@@ -24,11 +24,11 @@
 <resources>
 
     <!-- Toolbar and Bottombar Colors -->
-    <color name="toolbar_background">#20103B</color>
-    <color name="toolbar_icons">@color/solid_white</color>
+    <color name="toolbar_background">@color/solid_white</color>
+    <color name="toolbar_icons">@color/solid_black</color>
 
     <!-- Text Colors -->
-    <color name="toolbar_title">@color/solid_white</color>
+    <color name="toolbar_title">@color/solid_black</color>
```

Comment:
you don't need to define these three colors in the flavor color.xml because they are the same as in the main color.xml, so you may just delete them from the flavor specific color.xml files (embroidery, luna etc.)